### PR TITLE
announcement: FF Cloud features 3.1 Beta 2

### PR DESCRIPTION
**Pull request by ZJvandeWeg**
Soonish we'll allow folks to preview Node-RED 3.1 Beta 2. This blog post announces that.

See also: https://github.com/flowforge/admin/issues/149

### CODE REVIEW

File: src/blog/2023/03/node-red-3.1-beta-available-flowforge-cloud.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Node-RED 3.1 Beta 2 available on FlowForge Cloud
+subtitle: Test drive the next version of Node-RED
+description: To allow testing of the next version of Node-RED, FlowForge Cloud made the 3.1-beta-2 release available.
+date: 2022-03-10
+authors: ["zeger-jan-van-de-weg"]
+---
```
Comment (ZJvandeWeg) [inline src/blog/2023/03/node-red-3.1-beta-available-flowforge-cloud.md:7]:
```suggestion
authors: ["zeger-jan-van-de-weg"]
tags:
    - posts
---
```

File: src/blog/2023/03/node-red-3.1-beta-available-flowforge-cloud.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+To help our users test their flows in this new version, FlowForge allows you to
+copy an application through the "Duplicate Project" button in the settings. By
+clicking this button the version of Node-RED can be changed to the Beta release.
+We'd appreciate any feedback on the new Node-RED version, especially if there's
+backward incompatible changes. This way we can improve the release before the
+new version of Node-RED is deployed to production.
+
+While testing if your flows work as expected, you might enjoy using the improved
+editor context menu through right-clicking on the workspace, try out the flow
```
Comment (iskerrett) [inline src/blog/2023/03/node-red-3.1-beta-available-flowforge-cloud.md:26]:
```suggestion
editor context menu by right-clicking on the workspace, the flow
```

File: src/blog/2023/03/node-red-3.1-beta-available-flowforge-cloud.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+locking, or add images to flow descriptions.
```
Comment (iskerrett) [inline src/blog/2023/03/node-red-3.1-beta-available-flowforge-cloud.md:27]:
```suggestion
locking feature, and the ability to add images to flow descriptions.
```

Comment (iskerrett) [reply src/blog/2023/03/node-red-3.1-beta-available-flowforge-cloud.md:27]: It is not clear to me how you can use the last two features? Are these menu items?

File: src/blog/2023/03/node-red-3.1-beta-available-flowforge-cloud.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+
+We encourage all our users to take advantage of this opportunity to test their
+flows in Node-RED 3.1. This will not only help them identify any potential
+issues but also allow them to try out the new features and improvements in this
+version. We believe that this new version of Node-RED will make it even easier
```
Comment (iskerrett) [inline src/blog/2023/03/node-red-3.1-beta-available-flowforge-cloud.md:33]: The second sentence is repetitive. I would delete.

**Pull request closed without merging**